README.md

```diff
@@ -38,17 +38,20 @@ Table of Contents
 #### Running as nix flake
 `sbomnix` can be run as a nix flake from the `tiiuae/sbomnix` repository:
 ```bash
-$ nix run github:tiiuae/sbomnix#sbomnix
+# '--' signifies the end of argument list for `nix`.
+# '--help' is the first argument to `sbomnix`
+$ nix run github:tiiuae/sbomnix#sbomnix -- --help
 ```
 
-or from a locally cloned repository:
+or from a local repository:
 ```bash
 $ git clone https://github.com/tiiuae/sbomnix
 $ cd sbomnix
-$ nix run .#sbomnix
+$ nix run .#sbomnix -- --help
 ```
-Similarly, you can run `nixgraph` with `nix run github:tiiuae/sbomnix#nixgraph`.
-See the list of other supported flake targets with: `nix flake show`.
+Similarly, you can run `nixgraph` with something like `nix run github:tiiuae/sbomnix#nixgraph --  --help`
+
+See the full list of supported flake targets with: `nix flake show`
 
 #### Running as python script
 Running `sbomnix` as python script requires python3 and packages specified in [requirements.txt](./requirements.txt). You can install the required packages with:
```

Comment:
**P**ython script
**Python 3**

Reply:
Fixed in https://github.com/tiiuae/sbomnix/commit/b424d9b917e0de3151289f417ef927f3726e339a

Dropped the Python 3 requirement in documentation, which should be the default now. I don't think anyone should be using Python 2 at this point.